react/Viewer/helpers.js

```diff
@@ -32,3 +34,12 @@ export const isValidForPanel = ({ file }) => {
     isFromKonnector({ file })
   )
 }
+
+export const downloadFile = async ({ client, file, url }) => {
+  if (isEncrypted(file)) {
+    return client.collection('io.cozy.files').forceFileDownload(url, file.name)
+  }
+  return client.collection('io.cozy.files').download(file)
+}
+
+export const isFileEncrypted = file => isEncrypted(file)
```

Comment:
nit: let's test this method as well 🙏

Reply:
It's just a simple redirection, I'm not sure what the point of testing this?

Reply:
so what is the point of using this redirection, then?

Reply:
Because it seems easier to have a single helper file to import than to know which cozy-client files should be imported. Also, it avoids any ambiguity on the object name (`isFileEncrypted` vs `isEncrypted`)